servers/rendering/renderer_rd/forward_clustered/render_forward_clustered.h

```diff
@@ -54,6 +56,11 @@
 #define RB_TEX_VOXEL_GI SNAME("voxel_gi")
 #define RB_TEX_VOXEL_GI_MSAA SNAME("voxel_gi_msaa")
 
+#define RB_TEX_DR_ALBEDO SNAME("dr_albedo")
+#define RB_TEX_DR_NORMAL SNAME("dr_normal")
+#define RB_TEX_DR_POSITION SNAME("dr_position")
+#define RB_TEX_DR_ORM SNAME("dr_orm")
```

Comment:
Maybe use a different prefix for these textures. Without context I would have no idea what "dr" stands for.

Reply:
I'm open to suggestions but **D**eferred **R**endering seemed to fit nicely as an acronym..

Reply:
I would have just called it "deferred"